src/main/java/org/jenkinsci/plugins/mesos/MesosSlaveInfo.java

```diff
@@ -91,7 +89,9 @@ else if(maxExecutorsVal < minExecutorsVal) {
   private final String jnlpArgs;
   private final boolean defaultSlave;
   // Slave attributes JSON representation.
-  private final JSONObject slaveAttributes;
+  private String slaveAttributesString;
+  @Deprecated
+  private JSONObject slaveAttributes;
```

Comment:
mark as transient and send through readResolve() ?

Reply:
It _is_ sent through `readResolve`. It could be marked `transient`, though it makes no difference since we no longer write to it except to null it out.

Reply:
Better to make it transient then. I hope we eventually implement some JEP-200 static analysis, and such case would appear be on our radar for sure